rescoup/rc_m1/master/RC-01_MAST_HIST.DATA

```diff
@@ -0,0 +1,228 @@
+GRID
+------------------------------------------------------------------------------------------------
+
+--
+NEWTRAN
+
+--
+GRIDFILE
+ 0  1 /
+
+--
+GRIDUNIT
+METRES  /
+
+--
+INIT
+
+SPECGRID
+ 1 1 1 1 F /
+
+COORD
+  2000.0000  2000.0000  2000.0000   2000.0000  2000.0000  2002.5000
+  2100.0000  2000.0000  2000.0000   2100.0000  2000.0000  2002.5000
+  2000.0000  2100.0000  2000.0000   2000.0000  2100.0000  2002.5000
+  2100.0000  2100.0000  2000.0000   2100.0000  2100.0000  2002.5000
+/
+
+ZCORN
+  2000.0000  2000.0000  2000.0000  2000.0000  2002.5000  2002.5000
+  2002.5000  2002.5000 /
+
+
+PORO
+ 0.25 /
+  
+PERMX
+ 2100 /
+
+COPY
+ PERMX PERMY /
+ PERMX PERMZ /
+/
+
+MULTIPLY
+ 'PERMZ' 0.1 /
+/
+
+RPTGRID
+ 'ALLNNC' /
+
+------------------------------------------------------------------------------------------------  
+EDIT
+------------------------------------------------------------------------------------------------
+
+
+------------------------------------------------------------------------------------------------
+PROPS
+------------------------------------------------------------------------------------------------
+
+NOECHO
+
+
+INCLUDE
+'include/PVT-WET-GAS.INC' /
+
+INCLUDE
+ 'include/scal_mod2.inc' /
+
+
+FILLEPS
+
+------------------------------------------------------------------------------------------------
+REGIONS
+------------------------------------------------------------------------------------------------
+
+------------------------------------------------------------------------------------------------
+SOLUTION
+------------------------------------------------------------------------------------------------
+  
+RPTRST
+ BASIC=2 /
+
+EQUIL
+-- Datum    P     woc     Pc   goc      Pc  Rsvd  Rvvd  N
+2300.00   300.0  2300.0   0.0  1800.00  0.0   1     1   0 /   
+
+PBVD
+ 2000.0 100.0
+ 2300.0 100.0 /
+
+PDVD
+ 2000.0 100.0
+ 2300.0 100.0 /
+/
+
+
+------------------------------------------------------------------------------------------------
+SUMMARY
+-----------------------------------------------------------------------------------------------
+
+INCLUDE
+ 'include/summary.inc' /
+
+------------------------------------------------------------------------------------------------
+SCHEDULE
+------------------------------------------------------------------------------------------------
+
+
+TUNING
+ 1.0  5.0 /
+ /
+ /
+
+INCLUDE
+ 'include/b1_vfp_flowline.inc' /
+
+INCLUDE
+ 'include/d1_vfp_flowline.inc' /
+
+INCLUDE
+ './include/flowline_e1_vfp.inc' /
+
+
+GRUPTREE
+ 'PLAT-A' 'FIELD' /
+ 
+ 'MOD1'   'PLAT-A' /
+
+ 'B1_M'   'MOD1' /
+ 'D1_M'   'MOD1' /
+ 'C1_M'   'MOD1' /
+ 
+ 'E1_M'   'PLAT-A' /
+/
+
+SLAVES
+  'RES-1'  'RC-01_MOD1_HIST'   '*'  '../mod1'  4 /
+  'RES-2'  'RC-01_MOD2_HIST'   '*'  '../mod2'  1 /
+/
+
+--GRUPMAST
+-- 'D1_M' 'RES-1'  'MANI-D'  1*  / 
+-- 'B1_M' 'RES-1'  'MANI-B'  1*  / 
+-- 'C1_M' 'RES-1'  'MANI-C'  1*  / 
+-- 'E1_M' 'RES-2'  'E1'  1*  / 
+--/
```

Comment:
Do we expect to enable these at some point in the future?  If not, it might be better to just remove the block altogether so as to limit the risk of confusion.